macro/Youtube.py

```diff
@@ -4,4 +4,5 @@ def execute(macro, args):
   key = args.split()[0]
   if key.startswith('http://www.youtube.com/watch?v='):
     key = key[len('http://www.youtube.com/watch?v='):]
-  return macro.formatter.rawHTML('<object width="480" height="295"><param name="movie" value="http://www.youtube.com/v/%(key)s&hl=en_US&fs=1&hd=1"></param><param name="allowFullScreen" value="true"></param><param name="allowscriptaccess" value="always"></param><embed src="http://www.youtube.com/v/%(key)s&hl=en_US&fs=1&hd=1" type="application/x-shockwave-flash" allowscriptaccess="always" allowfullscreen="true" width="480" height="295"></embed></object>'%locals())
+  key = key.replace('&', '?', 1) # query params must start with ?
+  return macro.formatter.rawHTML('<center><iframe type="text/html" width="480" height="270" src="http://www.youtube.com/embed/%(key)s" frameborder="0" allowfullscreen /></center>'%locals())
```

Comment:
I tried that and it still cuts the page off after the Youtube entry. It's like the server stops sending html after the `</center>`.